src/services/import/excel/XLSImporterValidateObservations.ts

```diff
@@ -38,17 +38,38 @@ export default class XLSImporterValidateObservations extends AbstractImporter {
   private age: Repository<Age> = getRepository(Age);
 
   private checkEuRingCodes = async (excelData: DataCheck): Promise<void> => {
+    const statusCached = await this.status
```

Comment:
At first I can see that you have imported this cached repos. This entities are not cached by default. 
AS second, to find by id, you dont need querybuidel, simple `find` will be enough